README.md

```diff
@@ -6,24 +6,49 @@ This site is a collaborative project by the Signalen community and is meant to b
 
 ## Building, testing and deploying
 
-This site is built using [Gatsby](https://www.gatsbyjs.org/). First make sure you have [Node](https://nodejs.org/en/) installed on your machine. Then install the dependencies with the `npm install` command. After this you can use the `npm start` to start a local webserver and build/update the site.
+This site is built using [Gatsby](https://www.gatsbyjs.org/). First make sure you have [Node](https://nodejs.org/en/) installed on your machine. Then install the dependencies with the `npm install` command. After this you can use the `npm start` to start a local web server and build/update the site.
 
 The `master` branch of this repository is automatically deployed using Netlify.
 
+## How to add a blogpost to the website
+You can add a blogpost to the website by adding a Markdown document to the `pages/blog/` folder. The blog section of the website has multilingual support. Copy an existing file and edit it. Use the following filename convention: MM-DD-YYYY-filename.md
+
+Make sure the document includes the following frontmatter
+
+`---
+path: "/blog/02-11-2021-filename/"
+date: "2021-02-11"
+title: "Welkom op het Signalen blog"
+lang: nl
+template: default
+`
+Make sure you set the `path` and `lang` parameters. The lang is used to filter out the specific posts for a particular language.
+If you want to provide an English version of your blogpost make sure you prefix the path with `en` and your filename includes a language local using the following convention: `MM-DD-YYYY-filename.en.md`
```

Comment:
We should flip this around. Default should be English to be inline with the Standard for Public Code. If not we risk deviating even farther from meeting all the requirements.

Reply:
I do not agree because the community has explicitly decided that Dutch (for now) is the default language.  To move towards the standard we made the effort of also providing an English translation where visitors can acquire information.

Reply:
Yes, but that should only be a temporary exception for existing or ephemeral material. For new published material, we should gently steer towards the Standard in order to not make the gap impossible to close by making it wider over time. The community recently chose to be incubated towards compliance, so we should help them reach that goal sooner rather than later. And to be clear, I am not suggesting this should be a hard requirement, just that flipping the order in the instructions will get people in the mindset that English is the long term priority, but that Dutch of course is accepted too.

Reply:
Jan I agree but I think this is not the place to have that discussion. We already had this language discussion in product steering (more than once) if we do want to discuss it again I think that is the place to have that discussion. For now I'll leave it as it is.

Reply:
I am not suggesting opening a discussion to re-evaluate the current modus operandi. Just flipping the order of the instructions is not a policy change, just a framing to make everyone have the end goal in mind (and also future proofing this work).